controllers/managedmcg_controller_test.go

```diff
@@ -138,10 +138,35 @@ func newOcsCsvFake() opv1a1.ClusterServiceVersion {
 	return ocscsv
 }
 
+// Remove temporory files/folder or other data after finishing test.
 func cleanup() {
 	os.Remove(CustomerNotificationHTMLPath)
 }
 
+func miscTest(t *testing.T, rec *ManagedMCGReconciler) {
+	t.Helper()
+	t.Setenv("NOOBAA_CORE_IMAGE", "coreimage")
+	t.Setenv("NOOBAA_DB_IMAGE", "dbimage")
+
+	err := rec.lookupImages()
+	if err != nil || rec.images.NooBaaCore != "coreimage" || rec.images.NooBaaDB != "dbimage" {
+		t.Errorf("Can not find image files")
+	}
+
+	rec.managedMCG.UID = ""
+	_, err = rec.reconcilePhases()
+	if err != nil {
+		rec.Log.Error(err, "error reconciling ManagedMCG")
+	}
+
+	if err := rec.removeNoobaa(); err != nil {
+		t.Errorf("Error while removing Nooba: %v", err)
+	}
+	if err := rec.removeOLMComponents(); err != nil {
+		t.Errorf("Error while removing OLM: %v", err)
```

Comment:
```suggestion
		t.Errorf("Error while removing OLM: %w", err)
```
https://stackoverflow.com/a/61287626